packages/webapp/src/components/Form/Input/input.module.scss

```diff
@@ -81,15 +81,27 @@ input:focus::placeholder {
 .sm {
   margin-left: 8px;
 }
+
 .searchIcon {
-  color: var(--iconDefault);
+  width: 22px;
+  height: 22px;
   position: absolute;
-  font-size: 20px;
-  transform: translate(11px, 15px);
+  left: 11px;
+  top: 13px;
 }
+
+.searchClearIcon {
+  width: 17.255px;
+  height: 17.255px;
```

Comment:
I'm curious what's the actual lengths are on the screen...

Reply:
Apparently (AI says, ha) with enough screen resolution this level of precision can actually be represented! But mostly this is just my anxiety about not following Figma _exactly_ showing through 😉 I'll bump it down.